spark-bigquery-connector-common/src/test/java/com/google/cloud/spark/bigquery/SparkBigQueryConfigTest.java

```diff
@@ -166,6 +170,37 @@ public void testConfigFromOptions() {
     assertThat(config.getBigQueryClientConnectTimeout()).isEqualTo(10000);
     assertThat(config.getBigQueryClientReadTimeout()).isEqualTo(20000);
     assertThat(config.getBigQueryClientRetrySettings().getMaxAttempts()).isEqualTo(5);
+    assertThat(config.getArrowCompressionCodec()).isEqualTo(CompressionCodec.ZSTD);
+  }
+
+  @Test
+  public void testInvalidCompressionCodec() {
+    Configuration hadoopConfiguration = new Configuration();
+    DataSourceOptions options =
+        new DataSourceOptions(
+            ImmutableMap.<String, String>builder()
+                .put("table", "test_t")
+                .put("dataset", "test_d")
+                .put("project", "test_p")
+                .put("arrowCompressionCodec", "randomCompression")
+                .build());
+
+    try {
```

Comment:
Better to use `Assert.assertThrows()`, see [here for example](https://github.com/GoogleCloudDataproc/spark-bigquery-connector/blob/c2eeaf096c28e25af1929276c1d387d12c10fd3b/spark-bigquery-connector-common/src/test/java/com/google/cloud/spark/bigquery/SparkBigQueryProxyAndHttpConfigTest.java#L265)